includes/degree-functions.php

```diff
@@ -246,26 +234,88 @@ function get_degree_description_modern_layout() {
    *
    */
    function get_degree_application_deadline_modern_layout() {
-	 $application_deadline_first = get_field( 'application_deadline_first' );
-	 $application_deadline_second = get_field( 'application_deadline_second' );
+		$application_deadline_first = get_field( 'application_deadline_first' );
+		$application_deadline_second = get_field( 'application_deadline_second' );
 
-	 if( empty( $application_deadline_first ) && empty( $application_deadline_second ) ) return '';
- 
-	 ob_start();
-	 ?>
-	 <div class="bg-default">
-		<div class="container py-lg-3">
-			<div class="row my-lg-3">
-				<div class="col text-center text-uppercase">
-					<?php echo $application_deadline_first; ?><br>
-					<?php echo $application_deadline_second; ?>
+		if( empty( $application_deadline_first ) && empty( $application_deadline_second ) ) return '';
+
+		ob_start();
+		?>
+		<section aria-label="Application Deadline">
+			<div class=" d-block d-lg-none">
+				<div class="bg-primary text-center pt-5">
+					<h2 class="h4 text-uppercase font-condensed my-0">Application Deadline</h2>
+				</div>
+			<?php if( $application_deadline_first ) : ?>
+				<div class="text-uppercase text-center bg-primary py-4">
+					<?php echo $application_deadline_first; ?>
+				</div>
+			<?php endif; ?>
+			<?php if( $application_deadline_second ) : ?>
+				<div class="bg-primary py-1">
```

Comment:
This div & hr is showing up in md- down breakpoints, is that intentional? If not, you can add the `hidden-md-down` class here to hide it

Reply:
We are going to rework this section. We can address it then.